lib/Client.php

```diff
@@ -551,7 +552,6 @@ protected function sendAsyncInternal(RequestInterface $request, callable $succes
             $curl,
             $this->createCurlSettingsArray($request)
         );
-        curl_multi_add_handle($this->curlMultiHandle, $curl);
```

Comment:
Oups, I intented to delete this line but I double checked and it is actually needed by poll(). Seems like I got lost in the code flow. I'll change the PR soon.